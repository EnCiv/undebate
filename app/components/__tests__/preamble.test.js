import React from 'react'
import { shallow } from 'enzyme'

import Preamble from '../preamble'

describe('Preamble', () => {
  it('should render correctly', () => {
    const component = shallow(<Preamble />)

    expect(component).toMatchSnapshot()
  })
})
