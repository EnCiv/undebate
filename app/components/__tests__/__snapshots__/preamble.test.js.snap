// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preamble should render correctly 1`] = `
<Preamble
  classes={
    Object {
      "Preamble": "Preamble-Preamble-0-1-1",
      "agreed": "Preamble-agreed-0-1-2",
      "center": "Preamble-center-0-1-3",
    }
  }
/>
`;
